### PR TITLE
Feature/98 - Alert fixes

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

## Summary

- Removed warning and negative variants of Alert
- Centered content of an Alert
- Buttons are now taking all space in an Alert
- Alert is now a maximum of 460px
- Alert is now dismissible by default

### CODE REVIEW

File: packages/components/src/alert/src/Alert.css [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+    max-width: 50%;
+    flex-grow: 1;
+    flex-basis: 0;
+    min-width: 0;
 }
 
-.o-ui-alert-negative-icon {
-    color: var(--hop-danger-icon-weak);
+.o-ui-alert.o-ui-alert-dialog {
+    width: 28.75rem;
```
Comment (alexasselin008) [inline packages/components/src/alert/src/Alert.css:9]: Les dialogs ont des sizes. Est-ce qu'on devrait modifié `.o-ui-dialog-sm` pour mettre 28.75 au lieu de 34?

Comment (fraincs) [reply packages/components/src/alert/src/Alert.css:9]: alert a un fixed size

Comment (alexasselin008) [reply packages/components/src/alert/src/Alert.css:9]: Alert force déjà small au dialog.

c'Est juste que small c'est pas la bonne largeur

File: packages/components/src/dialog/src/Dialog.css [hunk 2]
```diff
@@ -28,6 +28,7 @@
     align-items: stretch;
     grid-template-areas: "image image" "aside aside";
     grid-template-rows: min-content auto;
+    width: 28.75rem;
```
Comment (alexasselin008) [inline packages/components/src/dialog/src/Dialog.css:31]: This doesn't seem right, why hardcode a with on the dialog

Comment (fraincs) [reply packages/components/src/dialog/src/Dialog.css:31]: Just removed when I noticed.

File: packages/components/src/alert/src/Alert.tsx [hunk 3]
```diff
@@ -158,11 +148,12 @@ export function InnerAlert({
 
     return (
         <Dialog
+            className="o-ui-alert o-ui-alert-dialog"
```
Comment (alexasselin008) [inline packages/components/src/alert/src/Alert.tsx:151]: scuse className doit être dans le merge aussi, pcq sinon si un user te passe un className le tiens se fait overwrité

File: packages/components/src/alert/src/Alert.css [hunk 4]
```diff
@@ -1,7 +1,29 @@
-.o-ui-alert-warning-icon {
-    color: var(--hop-warning-icon-weak);
+.o-ui-alert-button-group > .o-ui-button {
+    max-width: 50%;
```
Comment (alexasselin008) [inline packages/components/src/alert/src/Alert.css:2]: I'm pretty sure this alert modal can have 3 buttons. I'm guessing this style breaks with 3 buttons

Comment (fraincs) [reply packages/components/src/alert/src/Alert.css:2]: That one issue I raised with design regarding the current design, apparently we will never have 3 buttons.

Comment (alexasselin008) [reply packages/components/src/alert/src/Alert.css:2]: The code still support it at the moment.

Comment (fraincs) [reply packages/components/src/alert/src/Alert.css:2]: image

Forgot it was working with 3 buttons.

File: .changeset/perfect-pandas-accept.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (alexasselin008) [inline .changeset/perfect-pandas-accept.md:1]: delete this changeset file

File: .changeset/thirty-toys-prove.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+"@orbit-ui/transition-components": major
+---
+
+Alert UI fixes - Alert is now closable by default
```
Comment (alexasselin008) [inline .changeset/thirty-toys-prove.md:5]: Alert is now closable by default. This is potentially a breaking change if you we're not expecting the alert to be closable. 
To fix this, either add `dismissable={false}` to the Alert, or handle the `onClose` event handler

File: .changeset/thirty-toys-prove.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+"@orbit-ui/transition-components": major
+---
+
```
Comment (alexasselin008) [inline .changeset/thirty-toys-prove.md:4]: Mentionne que le dialog taille sm a réduit de taille aussi

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**